internal/sync/sync.go

```diff
@@ -179,32 +183,47 @@ func (s *Synchronizer) sync() error {
 func (s *Synchronizer) Status() *types.SyncStatus {
 	latestBlockSaved := float64(s.syncManager.GetLatestBlockSaved())
 
+	startingBlockHash := new(felt.Felt)
```

Comment:
This value is never used.

```suggestion
	var startingBlockHash *felt.Felt
```